specs/sharding/beacon-chain.md

```diff
@@ -223,7 +223,7 @@ class PendingShardHeader(Container):
     # Who voted for the header
     votes: Bitlist[MAX_VALIDATORS_PER_COMMITTEE]
     # Has this header been confirmed?
-    confirmed: bool
+    confirmed: boolean
```

Comment:
`bool` is the Python type -- is this pulling in an SSZ definition?

Reply:
yeah, boolean is a type in `ssz.ssz_typing`

Reply:
Yes. One reason behind the change is that `setup.py` fails, when there is `bool` in data classes.
```
  File "/mnt/c/Users/ericsson/IdeaProjects/eth2.0-specs/setup.py", line 379, in dependency_order_class_objects
    for item in [dep, key] + key_list[key_list.index(dep)+1:]:
ValueError: 'bool' is not in list
```
I assume that it's better to use the SSZ analogue (as it's the case in `Validator::slashed`, for example).